mainlyetcetera/rotten_tomatillos
# Feature/movie loader

## Is this a fix or a feature?
- Feature
## What is the change/fix?
- This is the loading components for both the mini movie posters and the movie details
- Components were built with React Content Loader
- Both conditional renders that await data from our fetch calls now render loaders until the data is fetched. 
## Where should the reviewer start?
- In MovieLoader.js and DetailsLoader.js
- Both have minimal styling as React Content Loader makes use of svgs
- Loading components are wrapped in divs because of the nature of svgs
- This should make it easy to style when it comes to making our design responsive
## How should this be tested?
- It helps to throttle network connection to slow 3G in dev tools to really see them. 
- The initial loader has a 1.5 second setTImeout because it was a bit jarring for them to flash for a split second
- Make sure you see loading screens for the main page and when you click a movie
## Screenshots(if appropriate)
## This PR is related to which issues:
Closes #25
